### PR TITLE
Implement UI and Logic

### PR DESCRIPTION
**Pull request by emmyobonyo**
- Created Medium Level Wire UI for the App.
- Implemented logic to allow a user to Add a Book.
- Implemented Logic to allow a user Remove  a Book.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  li.appendChild(author);
+  li.appendChild(removeBtn);
+  bookList.appendChild(li);
+  titleInput.value = '';
+  authorInput.value = '';
+  books.push(theBook);
+  localStorage.setItem('title', JSON.stringify(books));
+  localStorage.setItem('author', theBook.author);
+  removeBtn.addEventListener('click', function (e) {
+    bookList.removeChild(li);
```
Comment (mricanho) [inline index.js:34]: - [ ] [OPTIONAL] Break this function, remember that a function must do one thing and one thing only.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const authorInput = document.querySelector('.author-input');
+const addBtn = document.querySelector('.add-btn');
+
+function book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+let books = [];
+
```
Comment (mricanho) [inline index.js:13]: - [ ] Add a function to use Local Storage, data must be preserved in the browser's memory by using Local Storage, [check this link](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md)

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @emmyobonyo @yahyaelganyni1, 

You did a great job with this app, the linters are up and running and you did good commit messages🔥🔥 

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

almost there

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,76 +1,62 @@
...
+👤 **Yahya EL Ganayni**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/yahyaelganyni1 )
+- Twitter: [@twitterhandle](https://twitter.com/@crazy20046)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yahya-el-ganayni-a456115b/)
 
 👤 **Author2**
```
Comment (bellom) [inline README.md:42]: # [Optional]
Heads-Up: Instead of `Author2`, It would be nice to put the name of the Author right?
Kindly do this before merging 👍

**Review by bellom — APPROVED** [review]
Hi Team 👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**